packages/cnb_app_lifecycle/packaging

```diff
@@ -0,0 +1,27 @@
+set -e
+
+# create GOPATH
+mkdir -p ${BOSH_INSTALL_TARGET}/src
+mv * ${BOSH_INSTALL_TARGET}/src
+mv ${BOSH_INSTALL_TARGET}/src .
+
+DEST="$PWD/bin"
+mkdir -p ${DEST}
+
+source /var/vcap/packages/golang-*-linux/bosh/compile.env
+
+pushd src/code.cloudfoundry.org
+
+CGO_ENABLED=0 go build -o ${DEST}/ -ldflags "-s -w" -a -installsuffix static code.cloudfoundry.org/cnbapplifecycle/cmd/builder code.cloudfoundry.org/cnbapplifecycle/cmd/launcher
```

Comment:
[code.cloudfoundry.org/cnbapplifecylce](https://github.com/cloudfoundry/cnbapplifecycle/tree/31eb7113e33657bd7c9838372e8da51a5ea5c481) is missing a LICENSE. This would get flagged during the OSL process, can we make sure that there is a LICENSE for this repo?

Reply:
added in https://github.com/cloudfoundry/cnbapplifecycle/pull/15